### DISCUSSION TITLE
Is it possible to choose the path where i want to create a note everytime ?

### DISCUSSION THREAD

**Discussion by MaxiimeG** [marcusolsson/obsidian-projects#574] [Help]
Hello, so i need to create new note in the root folder for my table view but when i create a note in my calendar view i would want to create the note in a subfolder name "calendar" for example.

I now know i can view 2 differents folders in the same projects and that i can set where to create the new note but it seems it create it for all the views and not only one ?

thanks in advance

**Comment by Acylation** [comment]
Yeah the path of new note is a project-level config. Let's get a brief summary on how the hack works:

The new note is stored in the target folder, and the views will track all the folders, including the root folder and sub ones. Then, if you get your filters applied, the views are able to display only the correponding subfolder.

Due to the limitation of new note creation, the creating action in both views will lead to the same result, locating the new note according to the path. So the use case of assigning new note folder when creation in multi-folder project is somehow ignored, and you may need to manually reposition the notes in the file explorer of Obsidian. This could be a feature request to me.

**Comment by MaxiimeG** [comment]
haha i really do all in the wrong orders, i ask a question is a issues and make a feature request in discussion, okay so i should open a request feature

**Reply by Acylation** [reply]
Nevermind, the repo owner can easily convert an issue to discussion, as well as create an issue base on the discussion. We can just stay here and wait for a triage (lol

**Reply by MaxiimeG** [reply]
Okay, so i dont close this discussions and just chill ?

**Reply by Acylation** [reply]
Yeah that's enough.
